eng/validation/build-arcadewithrepo.ps1

```diff
@@ -37,6 +37,9 @@ $global:daysOfOldestBuild = if (-not $daysOfOldestBuild) { 3 } else { $daysOfOld
 $global:pushBranchToGithub = $pushBranchToGithub
 $global:azdoRepoName = if (-not $azdoRepoName) { "" } else { $azdoRepoName }
 
+Write-Host "##vso[task.setvariable variable=arcadeVersion;isOutput=true]${global:arcadeSdkVersion}"
+Write-Host "##vso[task.setvariable variable=repo;isOutput=true]${global:githubOrg}/${global:githubRepoName}"
```

Comment:
NIT: perhaps `qualifiedRepositoryName` would be better.

Reply:
Yep, that's better :)